vcluster/configure/vcluster-yaml/sleep-mode.mdx

```diff
@@ -96,22 +132,21 @@ The [resource exemption](#resource-exemption) feature keeps specifically configu
 </details>
 
 :::note
-Add the annotations to the vCluster workload, such as the `StatefulSet` or `Deployment` running the virtual cluster. You can dynamically ignore any request by adding the `X-Sleep-Mode-Ignore` header.
+Add the annotations to the vCluster workload, such as the StatefulSet or Deployment running the virtual cluster. You can dynamically ignore any request by adding the `X-Sleep-Mode-Ignore` header.
 :::
 
-## vCluster sleep mode compatibility with the platform
-
-You can configure sleep mode in a virtual cluster in two ways:
-
-- **Without an agent**: This scales down workloads (like pods) while keeping the control plane active. The cluster remains in a minimal state and can wake up when activity is detected.
-
-- **With an agent**: When connected to the platform using an agent, the agent fully manages sleep mode by shutting down the control plane. This reduces resource usage more and enables features like auto-deletion, which aren’t available without the agent.
+### Connect to the platform
 
-When a vCluster connects to the platform using an agent, the agent manages sleep mode actions by shutting down the control plane completely. This reduces resource usage and enables features like auto-deletion, which is not available when the vCluster is not connected to the platform using an agent.
+When you connect a vCluster that has sleep mode configured to the platform (via an agent), the platform agent automatically takes over sleep mode management. The vCluster stops managing its own sleep mode and the agent handles all sleep/wake operations.
 
-### Connect to the platform
+:::tip Transitioning from vCluster-managed to agent-managed sleep mode
+When a vCluster with sleep mode configured connects to the platform:
+- The platform reads the existing `sleepMode` configuration
```

Comment:
When referring to instances created by vCluster, use 'virtual cluster' instead of 'vCluster'.